services/old-docs-jujucharms-com.yaml

```
@@ -0,0 +1,45 @@
+---
+
+kind: Service
+apiVersion: v1  # See https://bit.ly/2KdOtrZ
+metadata:
+  name: old-docs-jujucharms-com
+spec:
+  selector:
+    app: old-docs.jujucharms.com
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: http
+
+---
+
+kind: Deployment
+apiVersion: extensions/v1beta1  # See https://bit.ly/2KdOtrZ
+metadata:
+  name: old-docs-jujucharms-com
+spec:
+  replicas: 5
+  template:
+    metadata:
+      labels:
+        app: old-docs.jujucharms.com
+    spec:
+      containers:
+        - name: old-docs-jujucharms-com
+          image: prod-comms.docker-registry.canonical.com/docs.jujucharms.com:${TAG_TO_DEPLOY}
```

This should be changed too

```suggestion
          image: prod-comms.docker-registry.canonical.com/old-docs.jujucharms.com:${TAG_TO_DEPLOY}
```